parser/scanner.l

```diff
@@ -557,6 +558,7 @@ sys_var		"@@"(({global}".")|({session}".")|{local}".")?{ident}
 {right}			return right
 {rollback}		lval.item = string(l.val)
 			return rollback
+{row}			return row
```

Comment:
row is an unreserved keyword. Please treat is as other unreserved keyword, like "rollback".